common/srv/server.go

```diff
@@ -203,15 +203,31 @@ func SetupLogger(prefix string, atomicLevel *zap.AtomicLevel, flags *flag.FlagSe
 	var lowPrioFile, highPrioFile zapcore.WriteSyncer
 	var openerr error
 	if lFlag := flags.Lookup("l"); lFlag != nil && lFlag.Value.(flag.Getter).Get().(string) != "" {
-		lowPrioFile, _, openerr = zap.Open(lFlag.Value.(flag.Getter).Get().(string))
+		f, err := os.OpenFile(lFlag.Value.(flag.Getter).Get().(string), os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
+		if err != nil {
+			return nil, fmt.Errorf("Unable to open file for logger: %v", err)
+		}
+		err = syscall.Dup2(int(f.Fd()), int(os.Stdout.Fd()))
+		if err != nil {
+			return nil, errors.New("Unable to redirect STDOUT")
+		}
+		lowPrioFile = zap.CombineWriteSyncers(f)
```

Comment:
I'm not sure this is actually necessary is it?

Reply:
I put this one in, that way if someone does some debugging with fmt.Println, they will show up in the logs.